Sources/Fuzzilli/Lifting/JavaScriptLifter.swift

```diff
@@ -101,6 +101,8 @@ public class JavaScriptLifter: Lifter {
 
         var w = JavaScriptWriter(analyzer: analyzer, version: version, stripComments: !options.contains(.includeComments), includeLineNumbers: options.contains(.includeLineNumbers))
 
+        var loopLabelStack = LabelStack(type: .loopblock)
```

Comment:
Can we `assert` at the end of the lifting that this stack is now empty? Just to catch any cases where we forget the `pop()`